app/helpers/proxmox_vm_cloudinit_helper.rb

```diff
@@ -41,4 +41,49 @@ def parse_server_cloudinit(args)
     end
     cloudinit_h
   end
+
+  def create_cloudinit_iso(vm_name, configs)
+    iso = File.join('/var/lib/vz/template/iso', "#{vm_name}_cloudinit.iso")
```

Comment:
should this be hard-coded here or somewhere else? (see how it is done for other plugins)

Reply:
Other plugins like libvirt also have hard-coded path for storage

Reply:
As a const ```
        def default_iso_dir
          "/var/lib/libvirt/images"
        end```